use helper methods with xp tracker page

byendan opened the pull request:
I have updated the view for the xp tracker so it takes advantage of helper methods. One is the method you wrote to style the min xp based on the ration between min/max xp. The other method uses the current level to decide what the max xp should be. 

I also added a database column for level to the display, and put in some sample data for that column(level 3).

I'm not sure how to go forward with deprecating the display at the moment, so we should follow up on that issue #1 Use xp_tracker page and deprecate display.

InfAmelia commented on app/helpers/pages_helper.rb:
```diff
@@ -1,2 +1,69 @@
+       severity
+     end
+
+     def max_xp(level)
```
Admittedly it's super confusing, but the actual Max XP per level is like this: 

2nd 300

3rd 900

4th 2,700

5th 6,500

6th 14,000

7th 23,000

8th 34,000

9th 48,000

10th    64,000

11th    85,000

12th    100,000

13th    120,000

14th    140,000

15th    165,000

16th    195,000

17th    225,000

18th    265,000

19th    305,000

20th    355,000

You were looking at "xp to next level", this is cumulative

http://www.enworld.org/forum/showthread.php?367079-5e-XP-Chart-Progression-Question/page2

byendan commented:
Made the requested changes for cumulative xp tracking. Created a party model to keep track of xp and level. Removed all files relating to the display, and removed it from the database. Added seed data to the database for a default party, with xp and level.

InfAmelia commented:
Looks good 👍
